docs/history.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+`2.0.0`_ (2017-05-16)
```

Comment:
Might as well remove the unused link for `Next Release` since it appears to have been removed.